src/fides/api/ops/schemas/connection_configuration/connection_secrets_email_consent.py

```diff
@@ -14,16 +17,9 @@ class AdvancedSettings(BaseModel):
     identity_types: IdentityTypes
 
 
-class ConsentEmailSchema(BaseModel):
-    """Schema to validate the secrets needed for the generic ConsentEmailConnector
+class ConsentEmailSchema(EmailSchema):
```

Comment:
Cleaning up by extending `EmailSchema` that already contains `third_party_vendor_name`, `recipient_email_address`, `test_email_address`

Reply:
The original `ConsentEmailSchema` was intended to be the more generic one that other connectors could extend. I am wondering if instead we should rename `ConsentEmailSchema` to `EmailSchema`, instead of introducing your new `EmailSchema`.

The main thing difference is that the `ConsentEmailSchema` allows **various identity types to be saved directly on the connector**.    For example, a connector might require that we have a "phone number" from the user in order to fire.  Otherwise, we shouldn't contact the third party vendor or send any other user identifiers to them.   The work with the Attentive connector only allows erasure email connectors to be created that pass on the user's email to the third party vendor.  I think we should build in more flexibility now to store the identifiers on the connector itself. 

We also had planned to put in UI around this once we could prioritize it, but for now we were just enforcing this in the connector's schema.  This is why we have that "advanced_settings" key.

![image](https://user-images.githubusercontent.com/9755598/224804283-21bee969-a1b2-4d88-a62b-16bbb15ca17e.png)

Reply:
Thanks for the background on ConsentEmailSchema. I originally created my own because I didn't see how the advanced settings would be relevant for an erasure email connector but when you mentioned wanting to save identity types directly on the connector it made sense. I renamed ConsentEmailSchema to EmailSchema and deleted the schema I created.

Reply:
Really appreciate the reorganization, looks great